services/libs/types/src/llm.ts

```diff
@@ -82,6 +82,14 @@ export const LLM_SETTINGS: Record<LlmQueryType, ILlmSettings> = {
       temperature: 0,
     },
   },
+  [LlmQueryType.MATCH_MAIN_GITHUB_ORGANIZATION_AND_DESCRIPTION]: {
+    modelId: LlmModelType.CLAUDE_3_5_SONNET,
```

Comment:
AFAIK V2 is only better and with the same pricing - why not use V2 here as well?